src/jit/compphases.h

```diff
@@ -22,6 +22,12 @@
 CompPhaseNameMacro(PHASE_PRE_IMPORT,             "Pre-import",                     "PRE-IMP",  false, -1)
 CompPhaseNameMacro(PHASE_IMPORTATION,            "Importation",                    "IMPORT",   false, -1)
 CompPhaseNameMacro(PHASE_POST_IMPORT,            "Post-import",                    "POST-IMP", false, -1)
+CompPhaseNameMacro(PHASE_MORPH1,                 "Morph - Starting",               "MORPH1",   false, -1)
```

Comment:
Could you make these names a bit more descriptive? e.g. `PHASE_MORPH_INIT`, `PHASE_MORPH_INLINE`, etc.